### PR TITLE
dapr_scheduler: Adds scheduler-volume flag

### PR DESCRIPTION
**Pull request by JoshVanL**
Expose `--scheduler-volume` to optionally specify the volume which scheduler writes to.

### CODE REVIEW

**Review by mikeee — DISMISSED** [review]
lgtm! Resolves the remaining scheduler init issue for me - #1416

**Review by yaron2 — APPROVED** [review]

File: cmd/init.go [hunk 1]
```diff
@@ -219,6 +224,7 @@ func init() {
 	InitCmd.Flags().String("network", "", "The Docker network on which to deploy the Dapr runtime")
 	InitCmd.Flags().StringVarP(&fromDir, "from-dir", "", "", "Use Dapr artifacts from local directory for self-hosted installation")
 	InitCmd.Flags().StringVarP(&imageVariant, "image-variant", "", "", "The image variant to use for the Dapr runtime, for example: mariner")
+	InitCmd.Flags().StringVarP(&schedulerVolume, "scheduler-volume", "", "", "Self-hosted only. Optionally specify a volume for the scheduler service data directory. By default, scheduler data is not persisted and not resilient to restarts without this flag.")
```
Comment (artursouza) [inline cmd/init.go:227]: We should have a volume by default instead of not persisting stuff.

Comment (mikeee) [reply cmd/init.go:227]: https://github.com/dapr/dapr/blob/7ea7016c0150e6a7b7bc7c30e34d56f0001c5e90/cmd/scheduler/options/options.go#L82

Agreed, unless the defaults are changed in both places. The mount should in its current state be created by default for the CLI 1.14.0-rc.4 release - in its current state a `dapr init` does not result in the scheduler running successfully.

**Comment by marcduiker** [comment]
@holopin-bot @JoshVanL Thanks Josh!

**Pull request merged**